pylhc/constants/calibration.py

```diff
@@ -0,0 +1,126 @@
+# IPS to be used to compute the calibraton factors
```

Comment:
Same as above. Some (short) docstring header and 
add this to the `doc/module/` as `.rst` (see there)